### DISCUSSION TITLE
I suggest moving tif_getimage_64.c and the modified tiffio.h to toonz/sources/image/tif rather than have it under third party.

### DISCUSSION THREAD

**Discussion by ghost** [opentoonz/opentoonz#6398] [Ideas]
There's a thread in the gitlab repo for libtiff about this and multiple times it was said it's definitely possible to have what we want and still use the unmodified system tiff library. I suggest that instead of relying on a modified libtiff, we just move tif_getimage_64.c and move the modified tiffio.h (under a new name like tiffio_new.h or something like that) to toonz/sources/image/tif. Any objections to that? Because it seems a way we can have our cake and eat it too. I think that just a handful of files would need to be updated to know the new paths and the new name for the tiffio.h header file. I already have that setup on my fork repo, so check it out and see what needs to be further changed.

https://github.com/jointri/opentoonz

**Comment by RodneyBaker** [comment]
For those interested...

@manongjohn has spend some considerable time updating third party libraries in the Tahoma2D fork.
This includes updating libtiff.

REF:  https://github.com/tahoma2d/tahoma2d/pull/1037
